libweb3jsonrpc/Eth.cpp

```diff
@@ -110,7 +110,10 @@ Json::Value Eth::eth_accounts() {
     return toJson( m_ethAccounts.allAccounts() );
 }
 
-string Eth::eth_blockNumber() {
+string Eth::eth_blockNumber( const Json::Value& request ) {
+    if ( !request.empty() ) {
+        BOOST_THROW_EXCEPTION( JsonRpcException( Errors::ERROR_RPC_INVALID_PARAMS ) );
```

Comment:
We need to return the same message as Ethereum does - "too many arguments, want at most 0"

Reply:
this is our standard error message for all json-rpc methods. should we change all of them?

Reply:
Let's raise a ticket about compatibility of error messages with EVM's and change them in separate PR's, then